engines/mtropolis/detection_tables.h

```diff
@@ -190,6 +195,52 @@ static const MTropolisGameDescription gameDescriptions[] = {
 				{ "TEXTWORK.R95", 0, "96346d39c4bb04f525edbf06ffe047e0", 148992 },
 				{ "EXPRMNTL.R95", 0, "aa0431c2be37e33883747c61d3e980ff", 108544 },
 				{ "MCURSORS.C95", 0, "47cf6abb95f3c43cdcbdf7ea1de3478d", 145920 },
+				{ "OBSIDIAN.MPL", 0, "4d557cd0a5f2311685d213053ebbd567", 116947911 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_DEMO | ADGF_UNSTABLE,
+			GUIO0()
+		},
+		GID_OBSIDIAN,
+		0,
+		0,
+	},
+
+	{ // Obsidian PC demo found on Multimedia Live (PC World) (v2.11, May 1997) [1996-10-03] [using 8.3 names]
```

Comment:
If it is using 8.3. games, will this entry work at all?

Reply:
Technically no, it'd require renaming the files. I'm trying to find a copy of the demo using the original names

Reply:
Then why not add those 8.3. name to the detection?